zio-http-gen/src/main/scala/zio/http/gen/openapi/Config.scala

```diff
@@ -19,20 +23,55 @@ final case class Config(
    *
    * TODO: in the future we can consider an enum instead of boolean for different aliased types.
    *       e.g: scala 3 opaque types, neotype, prelude's newtype, etc'…
+   */,
+  fieldsNormalizationConf: NormalizeFields, /*
+   * OpenAPI can declare fields that have unconventional "casing" in scala,
+   * like snake_case, or kebab-case.
+   * This configuration allows to normalize these fields.
+   * The original casing will be preserved via a @fieldName("<original-name>") annotation.
    */
 )
 object Config {
 
+  final case class NormalizeFields(
+    enabled: Boolean /*
```

Comment:
Block comments belong before not after what is commented. Anyhow, I think this should be comments for the config class
/**
     * 
     * @param enabled
     */